crates/biome_js_formatter/src/comments.rs

```diff
@@ -157,6 +153,29 @@ fn handle_typecast_comment(comment: DecoratedComment<JsLanguage>) -> CommentPlac
     }
 }
 
+
+fn handle_after_arrow_fat_arrow_comment(
```

Comment:
Could you add a documentation comment that explains what this function does? How it moves the comments and where

Reply:
Yeah it's great! thank you